rosa_learning/deploying_application_workshop/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,151 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-deployment"]
+= Deploying the OSToy application with Kubernetes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-deployment
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 23-JAN-2024
+
+Deploy the OSToy application by creating and storing the images for the front-end and back-end microservice containers in an image repository. Then create Kubernetes deployments to deploy the application. 
+
+== Retrieving the login command
+
+. If you are not logged in to the command line interface (CLI), access your cluster with the web console.
+
+. Click the dropdown arrow next to your login name in the upper right, and select *Copy Login Command*.
++
+image::4-cli-login.png[CLI login screen]
++
+A new tab opens. 
+
+. Select your authentication method.
+
+. Click *Display Token*.
+
+. Copy the command under *Log in with this token*. 
+
+. From your terminal, paste and run the copied command. If the login is successful, you will see the following confirmation message:
++
+[source,terminal]
+----
+$ oc login --token=<your_token> --server=https://api.osd4-demo.abc1.p1.openshiftapps.com:6443
+Logged into "https://api.myrosacluster.abcd.p1.openshiftapps.com:6443" as "rosa-user" using the token provided.
+
+You don't have any projects. You can try to create a new project, by running
+
+oc new-project <project name>
+----
+
+== Creating a new project
+
+=== Creating a new project using the CLI
+
+. Create a new project named `ostoy` in your cluster by running following command:
++
+[source,terminal]
+----
+$ oc new-project ostoy
+----
++
+.Example output
+[source,terminal]
+----
+Now using project "ostoy" on server "https://api.myrosacluster.abcd.p1.openshiftapps.com:6443".
+----
+
+. *Optional*: Create a unique project name by running the following command:
++
+[source,terminal]
+----
+$ oc new-project ostoy-$(uuidgen | cut -d - -f 2 | tr '[:upper:]' '[:lower:]')
+----
+
+=== Creating a new project using the web console
+
+. From the web console, click *Home -> Projects*. 
+
+. On the *Projects* page, click create *Create Project*.
++
+image::4-createnewproj.png[The project creation screen]
```

Comment:
We need to specify some things after doing this, at least a project name. https://docs.openshift.com/container-platform/4.17/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects